### DISCUSSION TITLE
Z3GateScheduleConfigurator

### DISCUSSION THREAD

**Discussion by jakstevns** [inet-framework/inet#985] [General]
![multicast](https://github.com/inet-framework/inet/assets/148611613/3f2b20ad-5c76-406e-a594-094009a4d76d)
The Z3GateScheduleConfigurator of gateScheduleConfigurator.configuration not support multicast ? I configured ST stream multicast, but it did not take effect. The configuration is attached

**Comment by levy** [comment]
IIRC the currently implemented gate schedulers don't support multicast streams.
